WordPress/Classes/ViewRelated/Reader/ReaderPostCellActions.swift

```diff
@@ -84,6 +84,15 @@ class ReaderPostCellActions: NSObject, ReaderPostCellDelegate {
 
         ReaderLikeAction().execute(with: post, context: context, completion: {
             cell.refreshLikeButton()
+            guard let origin = self.origin as? ReaderStreamViewController else {
+                return
+            }
+            // since a like status changed, let's refresh other tabs
+            WPTabBarController.sharedInstance().readerTabViewModel.invalidateCache()
```

Comment:
I think it would be nicer here if we could do something more decoupled, like posting a notification which the `ReaderTabViewModel` could listen out for.  I understand this may be an interim measure though.

Reply:
I agree that this design is not ideal, but I also don't particularly like to broadcast a notification for something this specific. I'll put some more thoughts on this and see if I can come up with a better design maybe with some injection..